synthesizer/src/process/mod.rs

```diff
@@ -102,6 +102,14 @@ impl<N: Network> Process<N> {
         Ok(())
     }
 
+    /// Removes a program from the process.
+    #[cfg(feature = "testing")]
+    #[inline]
+    pub fn remove_program(&mut self, program_id: &ProgramID<N>) -> Result<()> {
+        self.stacks.remove(program_id);
+        Ok(())
+    }
```

Comment:
```suggestion
    pub fn remove_program(&mut self, program_id: &ProgramID<N>) {
        self.stacks.remove(program_id);
    }
```
no need for the `Result`

Reply:
github seems to have glitched and duplicated my comment